### ISSUE TITLE
Examples of usage

### ISSUE THREAD

**Issue by nilov** [phpmentors-jp/workflower-bundle#5]
Hi! Thank's for library. But, I can't understand how it use. Can you provide some examles?

**Comment by iteman** [comment]
I will start to create an example Web application with Workflower as soon as possible :sweat_smile: 
And feel free to create an issue for your question.

**Comment by marek-l** [comment]
@iteman Hi, any news with some examples/documentation? :)

**Comment by iteman** [comment]
[The Quick Start Guide](https://github.com/phpmentors-jp/workflower/blob/master/docs/quick-start-guide.md) has been released  :sweat_smile:

**Comment by marek-l** [comment]
@iteman yes, but may be I have not enought experiences with BPMN, but I can not create working example according to Quick start guide. Do you have any demo example with functional classes?

thanks very much

**Comment by cherepanovic** [comment]
have the same problem, even after have read the quick start guide :/

**Comment by 77web** [comment]
I published example app here: https://github.com/77web/workflower-bundle-playground
It's still WIP, but very close to finish! :)
